client/src/helpers/Constants.js

```diff
@@ -2,7 +2,6 @@ export const MENU_ITEMS = [
   { text: "Find Food", link: "/organizations" },
   { text: "About Us", link: "/about" },
   { text: "Donate", link: "/donate" },
-  { text: "FAQs", link: "/faqs" },
```

Comment:
we want to keep this too

Reply:
Thanks Hannah, I reverted those changes and did it as suggessted.  😄